### PR TITLE
More HUDs

### PR DESCRIPTION
**Pull request by 1Sparky1**
Adds a bunch of new faction specific HUDs for both security (Variants of the non-glasses HUD) and medical (variants of the aviator and non-glasses HUD). These all should be coloured with their faction's palette (but I am really bad for messing that up somewhere).

### CODE REVIEW

**Comment by 1Sparky1** [comment]
!review

**Review by NonQueueingMatt — APPROVED** [review]

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]
look over all items and their name variable added in this PR and make sure `\The` is prefixed, so that it reads as "X throws the Zeng-Hu health scanner HUD!" and not "X throws Zeng-Hu health scanner HUD!".

File: code/modules/clothing/glasses/glasses.dm [hunk 1]
```diff
@@ -877,30 +922,29 @@ BLIND     // can't see anything
 	name = "Zavodskoi HUD aviators"
 	desc = "Zavodskoi security aviator glasses that can be switched between HUD and flash protection modes. They come with a built-in prescription overlay."
 	icon_state = "aviator_sec_zavod"
-	off_state = "aviator_sec_zavod_off"
+	item_state = "aviator_sec_zavod"
 
 /obj/item/clothing/glasses/sunglasses/sechud/aviator/pmc
 	name = "PMCG HUD aviators"
```
Comment (SleepyGemmy) [inline code/modules/clothing/glasses/glasses.dm:928]:
```suggestion
	name = "\improper PMCG HUD aviators"
```

File: code/modules/clothing/glasses/hud.dm [hunk 2]
```diff
@@ -26,6 +28,30 @@
 /obj/item/clothing/glasses/hud/health/is_med_hud()
 	return active
 
+/obj/item/clothing/glasses/hud/health/pmc
+	name = "PMCG health scanner HUD"
```
Comment (SleepyGemmy) [inline code/modules/clothing/glasses/hud.dm:32]:
```suggestion
	name = "\improper PMCG health scanner HUD"
```

File: code/modules/clothing/glasses/hud.dm [hunk 3]
```diff
@@ -26,6 +28,30 @@
...
+	desc = "A heads-up display in the colours of the Private Military Contracting Group."
+	icon_state = "healthhud_pmc"
+	item_state = "healthhud_pmc"
+
+/obj/item/clothing/glasses/hud/health/pmc/alt
+	name = "EPMC health scanner HUD"
```
Comment (SleepyGemmy) [inline code/modules/clothing/glasses/hud.dm:38]:
```suggestion
	name = "\improper EPMC health scanner HUD"
```

File: code/modules/clothing/glasses/hud.dm [hunk 4]
```diff
@@ -26,6 +28,30 @@
...
+	desc = "A heads-up display in Eridani Private Military Contracting colours."
+	icon_state = "healthhud_pmc_alt"
+	item_state = "healthhud_pmc_alt"
+
+/obj/item/clothing/glasses/hud/health/nt
+	name = "NanoTrasen health scanner HUD"
```
Comment (SleepyGemmy) [inline code/modules/clothing/glasses/hud.dm:44]:
```suggestion
	name = "\improper NanoTrasen health scanner HUD"
```

File: code/modules/clothing/glasses/hud.dm [hunk 5]
```diff
@@ -26,6 +28,30 @@
...
+	desc = "A heads-up display in the colours of the NanoTrasen Corporation."
+	icon_state = "healthhud_nt"
+	item_state = "healthhud_nt"
+
+/obj/item/clothing/glasses/hud/health/zeng
+	name = "Zeng-Hu health scanner HUD"
```
Comment (SleepyGemmy) [inline code/modules/clothing/glasses/hud.dm:50]:
```suggestion
	name = "\improper Zeng-Hu health scanner HUD"
```

File: code/modules/clothing/glasses/hud.dm [hunk 6]
```diff
@@ -47,14 +73,40 @@
...
 	item_state = "securityhud"
 	body_parts_covered = 0
 	var/global/list/jobs[0]
+	contained_sprite = TRUE
 
 /obj/item/clothing/glasses/hud/security/is_sec_hud()
 	return active
 
+/obj/item/clothing/glasses/hud/security/zavod
+	name = "Zavodskoi security HUD"
```
Comment (SleepyGemmy) [inline code/modules/clothing/glasses/hud.dm:87]:
```suggestion
	name = "\improper Zavodskoi security HUD"
```

File: code/modules/clothing/glasses/hud.dm [hunk 7]
```diff
@@ -47,14 +73,40 @@
...
+	desc = "A heads-up display in the colours of Zavodskoi Interstellar."
+	icon_state = "securityhud_zavod"
+	item_state = "securityhud_zavod"
+
+/obj/item/clothing/glasses/hud/security/pmc
+	name = "PMCG security HUD"
```
Comment (SleepyGemmy) [inline code/modules/clothing/glasses/hud.dm:93]:
```suggestion
	name = "\improper PMCG security HUD"
```

File: code/modules/clothing/glasses/hud.dm [hunk 8]
```diff
@@ -47,14 +73,40 @@
...
+	desc = "A heads-up display in the colours of the Private Military Contracting Group."
+	icon_state = "securityhud_pmc"
+	item_state = "securityhud_pmc"
+
+/obj/item/clothing/glasses/hud/security/pmc/alt
+	name = "EPMC security HUD"
```
Comment (SleepyGemmy) [inline code/modules/clothing/glasses/hud.dm:99]:
```suggestion
	name = "\improper EPMC security HUD"
```

File: code/modules/clothing/glasses/hud.dm [hunk 9]
```diff
@@ -47,14 +73,40 @@
...
+	desc = "A heads-up display in Eridani Private Military Contractor colours."
+	icon_state = "securityhud_pmc_alt"
+	item_state = "securityhud_pmc_alt"
+
+/obj/item/clothing/glasses/hud/security/idris
+	name = "Idris security HUD"
```
Comment (SleepyGemmy) [inline code/modules/clothing/glasses/hud.dm:105]:
```suggestion
	name = "\improper Idris security HUD"
```

File: code/modules/clothing/glasses/glasses.dm [hunk 10]
```diff
@@ -160,11 +160,43 @@ BLIND     // can't see anything
...
+	if(active)
+		icon_state = initial(icon_state)
+		item_state = initial(item_state)
+	else
+		icon_state = "[initial(icon_state)]_off"
+		item_state = "[initial(item_state)]_off"
+
+/obj/item/clothing/glasses/hud/health/aviator/pmc
+	name = "PMCG medical HUD aviators"
+	desc = "Sunglasses in the colours of the Private Military Contracting Group. They come with a blue-tinted HUD and a chrome finish."
```
Comment (SleepyGemmy) [inline code/modules/clothing/glasses/glasses.dm:173]:
```suggestion
	desc = "Sunglasses in Private Military Contracting Group colours. They come with a blue-tinted HUD and a chrome finish."
```

File: code/modules/clothing/glasses/glasses.dm [hunk 11]
```diff
@@ -160,11 +160,43 @@ BLIND     // can't see anything
...
+
+/obj/item/clothing/glasses/hud/health/aviator/pmc/alt
+	name = "EPMC medical HUD aviators"
+	desc = "Sunglasses in Eridani Private Military Contracting colours. They come with a blue-tinted HUD and a chrome finish."
+	icon_state = "aviator_med_pmc_alt"
+	item_state = "aviator_med_pmc_alt"
+
+/obj/item/clothing/glasses/hud/health/aviator/nt
+	name = "NanoTrasen medical HUD aviators"
+	desc = "Sunglasses in the colours of the NanoTrasen Corporation. They come with a blue-tinted HUD and a chrome finish."
```
Comment (SleepyGemmy) [inline code/modules/clothing/glasses/glasses.dm:185]:
```suggestion
	desc = "Sunglasses in NanoTrasen colours. They come with a blue-tinted HUD and a chrome finish."
```

File: code/modules/clothing/glasses/glasses.dm [hunk 12]
```diff
@@ -160,11 +160,43 @@ BLIND     // can't see anything
...
+	icon_state = "aviator_med_nt"
+	item_state = "aviator_med_nt"
+
+/obj/item/clothing/glasses/hud/health/aviator/zeng
+	name = "Zeng-Hu medical HUD aviators"
+	desc = "Sunglasses in the colours of Zeng-Hu Pharmaceuticals. They come with a purple-tinted HUD and a chrome finish."
```
Comment (SleepyGemmy) [inline code/modules/clothing/glasses/glasses.dm:191]:
```suggestion
	desc = "Sunglasses in Zeng-Hu Pharmaceuticals colours. They come with a purple-tinted HUD and a chrome finish."
```

File: code/modules/clothing/glasses/glasses.dm [hunk 13]
```diff
@@ -766,42 +801,51 @@ BLIND     // can't see anything
...
 
 /obj/item/clothing/glasses/sunglasses/sechud/big/zavod
 	name = "fat Zavodskoi HUDsunglasses"
 	desc = "Fat sunglasses in the colours of Zavodskoi Interstellar. They come with a red-tinted HUD."
 	icon_state = "bigsunglasses_hud_zavod"
+	item_state = "bigsunglasses_hud_zavod"
 
 /obj/item/clothing/glasses/sunglasses/sechud/pmc
 	name = "PMCG HUDsunglasses"
 	desc = "Sunglasses in the colours of the Private Military Contracting Group. They come with a blue-tinted HUD and a chrome finish."
```
Comment (SleepyGemmy) [inline code/modules/clothing/glasses/glasses.dm:820]:
```suggestion
	desc = "Sunglasses in Private Military Contracting Group colours. They come with a blue-tinted HUD and a chrome finish."
```

File: code/modules/clothing/glasses/glasses.dm [hunk 14]
```diff
@@ -766,42 +801,51 @@ BLIND     // can't see anything
...
 	icon_state = "sunhud_pmcg"
+	item_state = "sunhud_pmcg"
 
 /obj/item/clothing/glasses/sunglasses/sechud/pmc/alt
 	icon_state = "sunhud_pmcg_alt"
+	item_state = "sunhud_pmcg_alt"
 
 /obj/item/clothing/glasses/sunglasses/sechud/big/pmc
 	name = "fat PMCG HUDsunglasses"
 	desc = "Fat sunglasses in the colours of the Private Military Contracting Group. They come with a blue-tinted HUD and a chrome finish."
```
Comment (SleepyGemmy) [inline code/modules/clothing/glasses/glasses.dm:830]:
```suggestion
	desc = "Fat sunglasses in Private Military Contracting Group colours. They come with a blue-tinted HUD and a chrome finish."
```

File: code/modules/clothing/glasses/hud.dm [hunk 15]
```diff
@@ -26,6 +28,30 @@
 /obj/item/clothing/glasses/hud/health/is_med_hud()
 	return active
 
+/obj/item/clothing/glasses/hud/health/pmc
+	name = "PMCG health scanner HUD"
+	desc = "A heads-up display in the colours of the Private Military Contracting Group."
```
Comment (SleepyGemmy) [inline code/modules/clothing/glasses/hud.dm:33]:
```suggestion
	desc = "A heads-up display in Private Military Contracting Group colours."
```

File: code/modules/clothing/glasses/hud.dm [hunk 16]
```diff
@@ -26,6 +28,30 @@
...
+
+/obj/item/clothing/glasses/hud/health/pmc/alt
+	name = "EPMC health scanner HUD"
+	desc = "A heads-up display in Eridani Private Military Contracting colours."
+	icon_state = "healthhud_pmc_alt"
+	item_state = "healthhud_pmc_alt"
+
+/obj/item/clothing/glasses/hud/health/nt
+	name = "NanoTrasen health scanner HUD"
+	desc = "A heads-up display in the colours of the NanoTrasen Corporation."
```
Comment (SleepyGemmy) [inline code/modules/clothing/glasses/hud.dm:45]:
```suggestion
	desc = "A heads-up display in NanoTrasen colours."
```

File: code/modules/clothing/glasses/hud.dm [hunk 17]
```diff
@@ -26,6 +28,30 @@
...
+	icon_state = "healthhud_nt"
+	item_state = "healthhud_nt"
+
+/obj/item/clothing/glasses/hud/health/zeng
+	name = "Zeng-Hu health scanner HUD"
+	desc = "A heads-up display in the colours of Zeng-Hu Pharmaceuticals."
```
Comment (SleepyGemmy) [inline code/modules/clothing/glasses/hud.dm:51]:
```suggestion
	desc = "A heads-up display in Zeng-Hu Pharmaceuticals colours."
```

File: code/modules/clothing/glasses/hud.dm [hunk 18]
```diff
@@ -47,14 +73,40 @@
...
 
+/obj/item/clothing/glasses/hud/security/zavod
+	name = "Zavodskoi security HUD"
+	desc = "A heads-up display in the colours of Zavodskoi Interstellar."
+	icon_state = "securityhud_zavod"
+	item_state = "securityhud_zavod"
+
+/obj/item/clothing/glasses/hud/security/pmc
+	name = "PMCG security HUD"
+	desc = "A heads-up display in the colours of the Private Military Contracting Group."
```
Comment (SleepyGemmy) [inline code/modules/clothing/glasses/hud.dm:94]:
```suggestion
	desc = "A heads-up display in Private Military Contracting Group colours."
```

File: code/modules/clothing/glasses/hud.dm [hunk 19]
```diff
@@ -47,14 +73,40 @@
...
+
+/obj/item/clothing/glasses/hud/security/pmc/alt
+	name = "EPMC security HUD"
+	desc = "A heads-up display in Eridani Private Military Contractor colours."
+	icon_state = "securityhud_pmc_alt"
+	item_state = "securityhud_pmc_alt"
+
+/obj/item/clothing/glasses/hud/security/idris
+	name = "Idris security HUD"
+	desc = "A heads-up display in the colours of the Idris Incorporated."
```
Comment (SleepyGemmy) [inline code/modules/clothing/glasses/hud.dm:106]:
```suggestion
	desc = "A heads-up display in Idris Incorporated colours."
```

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]

**Review by SleepyGemmy — APPROVED** [review]

**Pull request merged**